### ISSUE TITLE
False positive about not used members

### ISSUE THREAD

**Issue by hez2010** [dotnet/roslyn#43942]
**Version Used**: 5.0.100-preview.4.20252.12

**Steps to Reproduce**:

1. Create a blazor client side project
2. Wrtie below code:

App.razor:
```xml
<Button @onclick="_ => _clicked = true"></Button>
```
App.razor.cs:
```csharp
partial class App
{
    private bool _clicked;
}
```

**Expected Behavior**:
No warnings and no quick fix suggestions. 

**Actual Behavior**:
Field 'App._clicked' is not used.

**Comment by CyrusNajmabadi** [comment]
@hez2010 are you still experiencing this?  And can you clarify what you mean by `Field 'App._clicked' is not used.`?  Are you getting a specific diagnostic message?  Can you show a screenshot?

**Comment by hez2010** [comment]
Seems that it's no longer an issue now.
